src/app/shared/header/case-header.html

```diff
@@ -1,3 +1,8 @@
-<h2 class="heading-large">
+<h2 *ngIf="!caseTitle.label; else case_title" class="heading-large">
```

Comment:
1) can we make this more self explanatory introducing a boolean method in the component: 
`displayCustomTitle()`
2) can we avoid using else inside the ngIf? I think this reads better:
```
<h2 *ngIf="!displayCustomTitle()" ...>
...
<ng-template *ngIf="displayCustomTitle()"...>

Reply:
Changed to make it more readable